src/Links/Link.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {
+  FunctionComponent,
+  ReactElement,
+  ReactNode,
+  useContext,
+  MouseEventHandler,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { Link as ReactLink } from 'react-router-dom';
+import { BaseTheme } from '../Theme';
+
+export interface LinkProps {
+  /** Text or components to be displayed as a representation of the link */
+  children: ReactNode;
+  /** A string that represents the link path */
+  to: string;
+  /** Function to call on click event */
+  onClick: MouseEventHandler;
```

Comment:
Since the underlying component is an `a`, we shouldn't have an `onClick` handler for it. Navigation events are already being handled, and we shouldn't allow Links to be used for non-navigation events.

Reply:
Removed through 25eeb3e